Challenge-4/Bot_1/src/helper.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { ethers, HandleTransaction } from "forta-agent";
+import { createAddress } from "forta-agent-tools";
+import { MockEthersProvider } from "forta-agent-tools/lib/test";
+import { provideHandleGovernanceTransaction } from "./agent";
+import { ASSET_INFO, CONFIGURATOR_PROXY } from "./constants";
+import { getCollateralAsset } from "./helper";
+
+let handleTransaction: HandleTransaction;
+describe("Helper test suite", () => {
+  let mockProvider: MockEthersProvider;
+  let provider: ethers.providers.Provider;
+  let Iface: ethers.utils.Interface = new ethers.utils.Interface(ASSET_INFO);
+  let getCollateralName = new ethers.utils.Interface([
+    "function name() view returns (string)",
+  ]);
+
+  const mockAssetTokenAddress = createAddress("0x10957");
+  beforeEach(() => {
+    mockProvider = new MockEthersProvider() as any;
+    provider = mockProvider as unknown as ethers.providers.Provider;
+    handleTransaction = provideHandleGovernanceTransaction(
```

Comment:
This doesn't seem to be actually used in the test cases, so no need to import and declare.

Reply:
Not sure I understood what the issue is. Is it the mockProvider?

Reply:
I meant `handleTransaction`. It doesn't actually seem to be used in the tests.